build.xml

```diff
@@ -106,30 +124,40 @@
           </fileset>
         </phpdoc2>
         -->
-        <exec command="php ${srcdir}/vendor/bin/phpDocumentor-${phpdoc_version}.phar --cache-folder=${builddir}/docs_cache --title=&quot;VuFindHarvest API Documentation&quot; -t ${builddir}/apidocs -d ${srcdir}/src" passthru="true" />
+        <exec executable="php" passthru="true">
+          <arg line="${srcdir}/vendor/bin/phpDocumentor-${phpdoc_version}.phar --cache-folder=${builddir}/docs_cache --title=&quot;VuFindHarvest API Documentation&quot; -t ${builddir}/apidocs -d ${srcdir}/src" />
+        </exec>
       </then>
     </if>
   </target>
 
   <!-- PHPUnit -->
   <target name="phpunit" description="Run tests">
-    <exec dir="${srcdir}" command="${phpunit_command} -dzend.enable_gc=0 --log-junit ${builddir}/reports/phpunit.xml --coverage-clover ${builddir}/reports/coverage/clover.xml --coverage-html ${builddir}/reports/coverage/" passthru="true" checkreturn="true" />
+    <exec dir="${srcdir}" executable="${phpunit_command}" passthru="true" checkreturn="true">
```

Comment:
Ere and I finished upgrading Phing to 3.0 in the main project today, and we discovered a problem: in some cases, the phpunit_command property gets set to an executable plus arguments, and that breaks the new system here. The workaround was to create a new "sh" property that specifies the Unix shell command, and then make _that_ the executable while putting everything else into the argument line.

Here's where the new property is set up at the top of the file:

https://github.com/vufind-org/vufind/blob/dev/build.xml#L3

And here is how things are put together in the phpunit commands:

https://github.com/vufind-org/vufind/blob/dev/build.xml#L369-L370

So, for example, these two lines would become:

```
<exec dir="${srcdir}" executable="${sh}" passthru="true" checkreturn="true">
  <arg line="-c &apos;${phpunit_command} -dzend.enable_gc=0 --log-junit ${builddir}/reports/phpunit.xml --coverage-clover ${builddir}/reports/coverage/clover.xml --coverage-html ${builddir}/reports/coverage/&apos;" />
```

(Note the `-c &apos;${phpunit_command} ... &apos;` wrapped around everything to make it a command to run with the shell).